README.md

```diff
@@ -41,49 +41,54 @@ The best part about [stamps](https://en.wikipedia.org/wiki/Stamp_%28object-orien
 
 ```js
 const mixin1 = {
-  componentDidMount() {
+  componentWillMount() {
     this.state.mixin1 = true;
   },
 };
 
 const mixin2 = {
-  componentDidMount() {
+  componentWillMount() {
     this.state.mixin2 = true;
   },
 };
 
-const component = stampit(React, {
+const Component = stampit(React, {
   state: {
     comp: false,
     mixin1: false,
     mixin2: false,
   },
 
-  someMethod() {
+  _onClick() {
+    return this.state;
+  },
+
+  componentWillMount() {
     this.state.comp = true;
   },
 
-  componentDidMount() {
-    this.someMethod();
+  render() {
```

Comment:
I know arrow functions handle autobinding, but I was not aware of a way to use arrow functions in this instance.